backend/backend/settings.py

```diff
@@ -40,6 +40,8 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'corsheaders',
+    'backend',
```

Comment:
I need the _backend_ in this list, otherwise I get the following error:

![image](https://user-images.githubusercontent.com/23021242/52272799-e08e8780-2915-11e9-8d31-261cf79b68ab.png)